docs/PCB_Information

```diff
@@ -0,0 +1,80 @@
+# BrewPi ESP8266 PCB Information
+
+## No LCD Support, no buzzer, with screw connectors, and through-hole components.
+
+[Purchase Boards](https://pcbs.io/share/zMGb8)  
+
+Eagle Files:
+
+*   TBD
+
+### Bill of Materials:
+
+1x [2-Pin 5mm Pitch Screw Terminal](https://www.aliexpress.com/item/100PCS-2-Pin-Screw-Terminal-Block-Connector-5mm-Pitch-Free-Shipping/32700056337.html)  
+1x [4-Pin 5mm Pitch Screw Terminal](https://www.aliexpress.com/item/100-pcs-4-Pin-Screw-blue-PCB-Terminal-Block-Connector-5mm-Pitch/32658656423.html) (Alternatively, 4x 2-pin terminals)  
+1x [10k 1/4 Watt Axial Resistor](https://www.aliexpress.com/item/100pcs-10k-ohm-1-4W-10k-Metal-Film-Resistor-10kohm-0-25W-1-ROHS/32577051768.html)  
+1x [RJ-11 Jack](https://www.aliexpress.com/item/RJ11-socket-outlet-90-degree-6-core-crystal-head-95001-6P6C-black-base-HXDD2/32685158354.html)  
+1x [WeMos D1 Mini ESP8266 board](https://www.aliexpress.com/store/product/D1-mini-Mini-NodeMcu-4M-bytes-Lua-WIFI-Internet-of-Things-development-board-based-ESP8266/1331105_32529101036.html)
+
+##   
+LCD Support with DuPont connectors, and surface-mount level converter components.
```

Comment:
## LCD Support with DuPont connectors, and surface-mount level converter components.